axon-server/administration/error-codes.md

```diff
@@ -0,0 +1,658 @@
+          is received by a node that is no longer leader.</p>
+        <p>Normally the node would forward the request to the leader, but if there
+          is a leader change during the processing of the join request this may happen.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-1300</td>
+      <td style="text-align:left">
+        <p><b>NO_SUCH_APPLICATION </b>
+        </p>
+        <p>This error is returned when any client sends any instruction with a specific
+          application name that has not been registered with Axon Server.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-1301</td>
+      <td style="text-align:left">
+        <p><b>NO_SUCH_NODE </b>
+        </p>
+        <p>This error is returned when any client sends any instruction with a specific
+          node name that has not been registered with Axon Server.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-1302</td>
+      <td style="text-align:left">
+        <p><b>CONTEXT_NOT_FOUND </b>
+        </p>
+        <p>This error is returned when any client sends any instruction with a specific
+          context that has not been registered with Axon Server.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-1304</td>
+      <td style="text-align:left">
+        <p><b>CONTEXT_EXISTS </b>
+        </p>
+        <p>This error is returned when there is an instruction to create a specific
+          context within Axon Server and it already exists.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-1400</td>
+      <td style="text-align:left">
+        <p><b>NO_AXONSERVER_FOR_CONTEXT</b>
+        </p>
+        <p>This error is returned when there is no Axon Server node available for
+          the specified context.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-1500</td>
+      <td style="text-align:left">
+        <p><b>AXONSERVER_NODE_NOT_CONNECTED</b>
+        </p>
+        <p>This<b> </b>error is returned when there is an instruction to create a
+          specific context within Axon Server and it already exists.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"><em>Input Errors </em>
+      </td>
+      <td style="text-align:left"></td>
+      <td style="text-align:left"></td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-2000</td>
+      <td style="text-align:left">
+        <p><b>INVALID_SEQUENCE</b>
+        </p>
+        <p>This<b> </b>error is returned when there is a gap between the sequence
```

Comment:
A bold space?

Reply:
I've seen this in more location actually...guessing it's a typo.

Reply:
I've seen this in more location actually...guessing it's a typo.